notes/cryptanalysis.tex

```diff
@@ -38,16 +205,56 @@ \section{Block Cipher Design and Cryptanalysis}
 \enm
 \end{theorem}
 
+\begin{proof}
+	Let's assume \(K[S_k] = 0\), this means for \(X = \left(\{M_i[S_m]\oplus C_i[S_c]\}_{i=1}^q\right)\), so a majority of sampled values resulted in a 0. \newline
+
+	\[ E[X] = q*(\frac{1}{2} + \epsilon) \]
+	\[ U = \Maj\left(\{M_i[S_m]\oplus C_i[S_c]\}_{i=1}^q\right) \]
 
-\begin{theorem}[Chernoff bounds]
-Let $X = \sum_{i=1}^n X_i$, all $X_i$ independent and where $X_i = 1$ with probability $p_i$ and $X_i = 0$
-with probability $1-p_i$. Let $\mu = \Ex[X]$. Then  
-\begin{align}
-  \Prob{X \ge (1 + \delta)\mu} &\le e^{-\frac{\delta^2}{2+\delta}\mu}\\
-  \Prob{X \le (1 + \delta)\mu} &\le e^{-\frac{\delta^2}{2}\mu}
-\end{align}
-\end{theorem}
 
+	\begin{theorem}[Chernoff bounds]
```

Comment:
should probably have some text before this to explain why you're introducing Chernoff bounds here. otherwise it comes out of nowhere